devbox.lock

```diff
@@ -1,6 +1,9 @@
 {
   "lockfile_version": "1",
   "packages": {
+    "github:NixOS/nixpkgs/nixpkgs-unstable": {
```

Comment:
We'll have to make sure everyone is on 0.14.0 of devbox as I don't know what happens if you haven't updated?